src/index.html

```diff
@@ -7,6 +7,339 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo">
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
```

Comment:
```suggestion

      </div>
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```
let's format code according to checklist 
it will have better readability
<img width="954" alt="image" src="https://user-images.githubusercontent.com/102411130/222067894-578ae01c-9f7d-419a-88b8-0c8fea06f4c1.png">